src/components/IconDropdown/_icon-dropdown.scss

```diff
@@ -0,0 +1,164 @@
+@import '~carbon-components/scss/components/dropdown/dropdown';
+
+@import '../../globals/vars';
+
+.#{$prefix}--list-box__menu-item__option {
+  padding: 0;
+}
+
+.#{$iot-prefix}--dropdown__footer {
+  @include box-shadow();
+
+  background-color: $ui-01;
+  z-index: z('dropdown');
+  width: 100%;
+
+  &-content {
+    padding: $spacing-03;
+  }
+}
+
+.#{$iot-prefix}--dropdown__selection-buttons {
+  > .#{$prefix}--list-box__field {
+    padding: 0 $spacing-09 0 $spacing-05;
+  }
+
+  > .#{$prefix}--list-box__menu {
+    outline-style: none;
+    border: none;
+    transition: initial;
+    box-shadow: none;
+
+    > .#{$prefix}--list-box__menu-item--active {
+      background: transparent;
+
+      > .#{$prefix}--list-box__menu-item__option {
+        > .#{$iot-prefix}--dropdown__image-button.bx--btn.bx--btn--icon-only.bx--tooltip__trigger:hover::before {
```

Comment:
The specificity of these selectors and =>3 levels deep nesting concerns me. The more specific, the more likely they are to break with future versions of Carbon.

Do the uses of the child combinator `>` in this file require that or could we use the less specific descendant combinator? Could we apply classes to the button's tooltip to prevent the long selector chain?

Completely understand if not - sometimes these things are somewhat unavoidable. Any improvements we can make would be solid!

Reply:
This breaking has been a worry for me as well, I will take another dive through this tonight, but I was trying to be as specific as possible to not screw up the other dropdowns in the codebase. Will comment here on my findings